app/templates/gulpfile.js

```diff
@@ -17,9 +17,13 @@ var production = false,
       scripts: 'app/scripts/**/*.js',<% } %>
       images:  'app/images/**/*.{png,gif,jpg,jpeg,svg}',
       fonts:   'app/fonts/**/*.{eot*,otf,svg,ttf,woff}',
-      vendor:  'vendor'
+      vendor:  'vendor',
+      styleguide: 'app/_components/**/*',
+      colors: 'app/_colors/**/*'
     };
 
+var replace = require('gulp-replace');
```

Comment:
This `require` is unnecessary. The `gulp-replace` plugin is already [included in the project](https://github.com/centresource/generator-playbook/blob/v2.0.1/app/templates/package.json#L20) and loaded via [`gulp-load-plugins`](https://github.com/centresource/generator-playbook/blob/v2.0.1/app/templates/gulpfile.js#L5). You have access to it via `$.replace()`.